components/container-cards/category-card-dark.config.yml

```diff
@@ -0,0 +1,9 @@
+collated: true,
+status: 'prototype'
```

Comment:
I think you should flag it as "wip", a prototype should not be implemented anywhere until it has moved on to be either wip or ready.